src/main/java/core/gui/comp/panel/DoubleLabelPanel.java

```diff
@@ -26,7 +26,7 @@ private void addLabels() {
         layout.setVgap(0);
         setLayout(layout);
         // Fix right label width to avoid components moving when values change.
-        var width = 60;
+        var width = UserParameter.instance().fontSize * 5;
```

Comment:
Not sure whether this helps (actually, most definitely not here, but maybe in other classes), but `stringWidth` in `FontMetrics` can help finding the size of a string in a given font. See for example [here](https://github.com/ho-dev/HattrickOrganizer/blob/master/src/main/java/core/icon/TextIcon.java#L33).

Reply:
Good idea for future refactorings. For time being, i only wanted to replace any constant by font size related values.